internal/testutil/acc/atlas.go

```diff
@@ -0,0 +1,39 @@
+package acc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/atlas-sdk/v20231115007/admin"
+)
+
+func createProject(tb testing.TB, name string) string {
+	tb.Helper()
+	orgID := os.Getenv("MONGODB_ATLAS_ORG_ID")
+	require.NotNil(tb, "Project creation failed: %s, org not set", name)
+	params := &admin.Group{Name: name, OrgId: orgID}
+	resp, _, err := ConnV2().ProjectsApi.CreateProject(context.Background(), params).Execute()
+	require.NoError(tb, err, "Project creation failed: %s, err: %s", name, err)
+	id := resp.GetId()
+	require.NotEmpty(tb, id, "Project creation failed: %s", name)
+	return id
+}
+
+func deleteProject(id string) {
+	_, _, err := ConnV2().ProjectsApi.DeleteProject(context.Background(), id).Execute()
+	if err != nil {
+		fmt.Printf("Project deletion failed: %s, error: %s", id, err)
+	}
+}
+
+func projectID(tb testing.TB, name string) string {
+	tb.Helper()
+	SkipInUnitTest(tb)
+	resp, _, _ := ConnV2().ProjectsApi.GetProjectByName(context.Background(), name).Execute()
+	id := resp.GetId()
+	require.NotEmpty(tb, id, "Project name not found: %s", name)
```

Comment:
can you make sure all asserions where you do string formatting end with `f` for format, other wise is a concatenation

Reply:
I think they do the same, e.g.:

```
a := "hello"
	require.Empty(t, a, "a is not empty: %s, watchout", a)

Error:      	Should be empty, but was hello
        	Test:       	TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchive_basic
        	Messages:   	a is not empty: hello, watchout
```

the signature is: func Empty(t TestingT, object interface{}, **msgAndArgs** ...interface{}) {

example of implementation:
```
//	assert.NotNilf(t, err, "error message %s", "formatted")
func NotNilf(t TestingT, object interface{}, msg string, args ...interface{}) bool {
	if h, ok := t.(tHelper); ok {
		h.Helper()
	}
	return NotNil(t, object, append([]interface{}{msg}, args...)...)
}
```

Reply:
i think it's just to make sure that the first param is a string (with the msg)